system/src/system_task.cpp

```diff
@@ -467,6 +467,26 @@ void manage_ble_prov_mode() {
 #endif
 }
 
+extern "C" void system_isr_task_queue_free_memory(uint8_t *ptrToFree) {
```

Comment:
`void*` argument, not `uint8_t*`

Reply:
Let's also not use this directly in the RTL SDK, perhaps. On second thought, let's mark `_freertos_mfree` as weak instead, revert all the changes, and reimplement it with system_isr_task_queue_free_memory()` when interrupts are disabled inside `rtl_sdk_support.cpp`.